doc/source/dev/development_environment.rst

```diff
@@ -194,9 +194,9 @@ That also takes extra arguments, like ``--pdb`` which drops you into the Python
 debugger when a test fails or an exception is raised.
 
 Running tests with `tox`_ is also supported.  For example, to build NumPy and
-run the test suite with Python 3.7, use::
+run the test suite with Python 3.9, use::
 
-    $ tox -e py37
+    $ tox -e py39
```

Comment:
For follow up: I wonder if this works. Should we add a tox CI run?

Reply:
I have good experience with using tox to manage the test environments and in a CI provider agnostic way, and as a nice bonus it also provides an easy way to reproduce the setup locally to debug specific failures. 
Now, I'm not sure how useful any of this would be for numpy, your test suite is quite complex as is and a lot of it may be even architecture-dependent and thus not reproducible on a local setup.

Anyway, if you think it's useful to have the tox file around, the it may make sense to rework one of the test jobs to pick up the env from here.